### DISCUSSION TITLE
Control OpenAPI version when exporting?

### DISCUSSION THREAD

**Discussion by rycee** [cue-lang/cue#1560] [Q&A] [answered]
Hello,

Looking at the CUE code it seems like it is possible to export to both OpenAPI v3.0.0 and v3.1.0. But I haven't been able to find any way to control the version from the commandline tool? Have I missed anything?

My goal is to export a file matching OpenAPI v3.1.0.

Thanks!

**Comment by mpvl** [comment, marked as answer]
I don't think that is supported. It probably should be supported through the "filetypes" mechanism, which supports adding flags (currently unused).

**Reply by rycee** [reply]
Thanks! Until the filetypes solution is available I'll continue using my workaround using jq to mold the v3.0.0 output into something sufficiently close to v.3.1.0 🙂
